client-spark/shuffle-manager-2/src/main/scala/org/apache/spark/shuffle/rss/RssShuffleFallbackPolicyRunner.scala

```diff
@@ -31,7 +31,8 @@ class RssShuffleFallbackPolicyRunner(sparkConf: SparkConf) extends Logging {
   def applyAllFallbackPolicy(dependency: ShuffleDependency[_, _, _],
```

Comment:
`dependency.partitioner.numPartitions` seems enough, could you please help to optimize this？Thanks

Reply:
Yes, I have retrieved the param [dependency] from this function